docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -407,12 +407,16 @@ the git revision for modules by setting a module manually with
 by specifying the `auto_update` option (as described below) to use the
 latest revision available for a module.
 
+Module fields support the expansion of variables defined in the [Variables](Project-and-Distro-Settings.md#variables)
```

Comment:
Can we add that all fields (except auto_update) can be expanded (maybe under the Fields section) and also indicate that these are expanded at version creation time?